### PR TITLE
[Rdu-2023] added sponsor info and speaker submission link

### PR DESCRIPTION
**Pull request by normalfaults**
added sponsor info and speaker submission link

### CODE REVIEW

**Comment by glasnt** [comment]
Hi! Can the prospectus please be added to https://github.com/devopsdays/devopsdays-assets thanks!

File: data/events/2023-raleigh.yml [hunk 1]
```diff
@@ -14,11 +14,11 @@ startdate: 2023-04-12  # The start date of your event. Leave blank if you don't
 enddate: 2023-04-13 # The end date of your event. Leave blank if you don't have a venue reserved yet.
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
+cfp_date_start: 2022-10-01  # start accepting talk proposals.
+cfp_date_end: 2023-01-20  # close your call for proposals.
```
Comment (glasnt) [inline data/events/2023-raleigh.yml:18]: Your form says January 15, 2023  but this says Jan 20.

**Comment by glasnt** [comment]
@normalfaults friendly ping to update this PR based on comments

**Comment by normalfaults** [comment]
> @normalfaults friendly ping to update this PR based on comments

Sorry was sick this week and didn’t see the reply. Will update after reviewing your comments. Thank you.

File: data/events/2023-raleigh.yml (hunk 1 above)
Comment (normalfaults) [reply data/events/2023-raleigh.yml:18]: Form has been updated to 1/20/23

File: data/events/2023-raleigh.yml [hunk 2]
```diff
@@ -14,11 +14,11 @@ startdate: 2023-04-12  # The start date of your event. Leave blank if you don't
...
 cfp_date_announce:  # inform proposers of status
 
-cfp_link: "" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+cfp_link: "https://docs.google.com/forms/d/e/1FAIpQLScM9ki8DOJqmcRSKhXVgk_pb5dXfFra5ECsN1tlXO-UcF3xNQ/viewform?usp=sf_link" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
```
[the remark this answers is not in the document]
Comment (normalfaults) [reply data/events/2023-raleigh.yml:21]: Form language updated to reflect optional submission.

**Comment by normalfaults** [comment]
> Hi! Can the prospectus please be added to https://github.com/devopsdays/devopsdays-assets thanks!
> 
> >  GitHub**[GitHub - devopsdays/devopsdays-assets: Contains binary/static files used by devopsdays events, but not served up via devopsdays.org](https://github.com/devopsdays/devopsdays-assets)**Contains binary/static files used by devopsdays events, but not served up via devopsdays.org - GitHub - devopsdays/devopsdays-assets: Contains binary/static files used by devopsdays events, but not...

Asset PR here: https://github.com/devopsdays/devopsdays-assets/pull/141

**Pull request merged**